tests/publiccloud/download_repos.pm

```diff
@@ -27,20 +27,19 @@ sub run {
     my ($self, $args) = @_;
     select_host_console();    # select console on the host, not the PC instance
 
-    # Skip maintenance updates. This is useful for debug runs
-    my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', 0);
+    assert_script_run("mkdir ~/repos");
 
-    # Trigger to skip the download to speed up verification runs
-    if ($skip_mu) {
+    if (get_var('PUBLIC_CLOUD_SKIP_MU', 0)) {
```

Comment:
```suggestion
    if (get_var('PUBLIC_CLOUD_SKIP_MU')) {
```

Reply:
well IMHO there is no difference , I mean I don't mind but just out of curiosity what reasoning do you have  ? :)

Reply:
no diff, but the default value is not needed for the condition.